### DISCUSSION TITLE
#59 Earth

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#164] [Q&A] [answered]
```
<p><p><p><c></c><c><d><style>
*>*{background:radial-gradient(1q,#4F77FF,4.7em,#191919)}
p{height:10;margin:105 0-75;font:0 a}
c{padding:96 ;border-radius:50%;background:0;border:10px solid#191919;position:fixed;
margin:-141 25}
c+c{margin:-141 147}
d{padding:90 5;margin:0-66}
```
endless 99.9% I can not make it work ... , any tip?

**Comment by Beowolve** [comment, marked as answer]
Try:
`border-radius:49.9%` :wink:

I need to optimize that as well, guess now I have a reason for looking at it again ;).

**Reply by rafehimself** [reply]
I hate that bug 💩

**Reply by RazvanManolache** [reply]
thanks beo, this has been most helpful

**Reply by verberg** [reply]
I have this problem too, border-radius:49.9% is not working. Have you any idea?
<p o><p><p x><style>*{background:#191919} p{width:190;position:fixed;aspect-ratio:1;background:#0000;border-radius:50%;border:10px solid#191919;top:29;left:35}[x]{left:155}[o]{width:150;top:49;left:115;border-radius:50%;background:linear-gradient(90deg,#0000 70px,0,#191919 80px,#0000 80px),repeating-linear-gradient(#4F77FF 0 30px,#0000 30px 40px
